### PR TITLE
[Feature] Add an indicator called errorRowsAfterResumed to distinguish between …

### PR DESCRIPTION
**Pull request by Henry2SS**
…totalErrorRows(called errorRows) and errorRowsAfterResumed.

## Proposed changes

When an routine load job's currentErrorRows is greater than maxErrorNum, the job will be paused. Users can use `resume routine load for xxx;` to make the job be running again.
Then, there will be a problem, the indicator called `errorRows` won't be reset to `0`. But as `totalErrorRows`, it is useful. So we need to add an indicator called  errorRowsAfterResumed to distinguish  between totalErrorRows(called `errorRows`) and `errorRowsAfterResumed`. We can get useful infomation through `errorRowsAfterResumed` , and determine whether some Propterties are reasonable or not.
`errorRowsAfterResumed` can directly show current errorRows, also at the time of the job's state is changed to Paused.  It can't just use the variable called `currentErrorRows`, cuz `currentErrorRows` will be reset to `0` after the job's state changed.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality) #6091 
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
The new added `errorRowsAfterResumed` is not persisted.
So If FE restart, this info is lost and will be reset to 0,  is it acceptable?

**Comment by Henry2SS** [comment]
> The new added `errorRowsAfterResumed` is not persisted.
> So If FE restart, this info is lost and will be reset to 0, is it acceptable?

Made `errorRowsAfterResumed` be persisted. Thx for your reminding.

**Review by morningman — APPROVED** [review]
LGTM

**Review by morningman — CHANGES_REQUESTED** [review]
Better refactor the statistic

**Comment by Henry2SS** [comment]
> Better refactor the statistic

**Comment by Henry2SS** [comment]
> Better refactor the statistic

Done.

File: fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java [hunk 1]
```diff
@@ -1436,15 +1420,20 @@ public void write(DataOutput out) throws IOException {
         out.writeLong(pauseTimestamp);
         out.writeLong(endTimestamp);
 
+        /*
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java:1423]: Remove these comment

Comment (Henry2SS) [reply fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java]: Finished. And I add some comments to explain the indicators called `errorRowsAfterResumed`.

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**